eta-company-calendar-frontend/src/app/shared/api-connectors/AuthApiConnector.ts

```diff
@@ -0,0 +1,33 @@
+import { HttpHeaders, HttpParams } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { AuthResponse } from '../models/auth-response.model';
+import { AbstractApiConnector } from './AbstractApiConnector';
+
+export class AuthApiConnector extends AbstractApiConnector {
+  protected readonly apiRoute: string = `${this.apiBaseUrl}/oauth/token`;
+
+  public requestToken(email: string, password: string): Observable<AuthResponse> {
+    const headers = new HttpHeaders({
+      'Authorization': 'Basic ' + btoa('fooClientIdPassword:secret'),
+      'Content-Type': 'application/x-www-form-urlencoded'
+    });
```

Comment:
ezek a headerek tuti kellenek?

Reply:
Az authorization-re szükség van, mert a backend azzal azonosítja ba a klienst. Enélkül 401-et kapa vissza.
Content type nélkül kipróbáltam postmanben, és úgy is beengedett, de benne hagynám mert a dokumentációt úgy értelmeztem, hogy ebben a formátumban kell küldeni.
https://tools.ietf.org/html/rfc6749#section-4.4.2